docs/config/README.md

```diff
@@ -148,15 +148,15 @@ min_time = 4
 ## Directory
 
 The `directory` module shows the path to your current directory, truncated to
-three parent folders. Your directory will also be truncated to the root of the
-git repo that you're currently in.
+three parent folders.
```

Comment:
```suggestion
three parent folders. Your directory will also be truncated to the root of the
git repo that you're currently in.
```

Reply:
Our usual policy is to describe the behavior with all the default options. You can add something to the line (e.g. "can be toggled with `truncate_to_repo`") if you're worried about people not realizing that this can be configured.